web/docs/getting-started.md

```diff
@@ -12,8 +12,8 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 ## 1. Requirements
 
 You need to have `node` (and `npm`) installed on your machine and available in `PATH`.
-- `node`: >=16.11.0 and <=16.15.0
-- `npm`: >= 8.0.0 and <= 8.5.5
+- `node`: >=16.11.0
+- `npm`: >= 8.0.0
```

Comment:
From what I understood (and what the `ci.yml` file specifies), we no don't require node to be `>=16.11.0`. It's enough to have `16.x.x`.

The specifier `>=16.0.0` would also be incorrect since we don't allow `17.0.0` and higher. The best way to specify the version is either with `^16` or `16.x.x`. Perhaps it's easiest if we just took the version we had before [Martin's recent revision (old version on the left)](https://github.com/wasp-lang/wasp/commit/62f0a7693740aebdfdc094419bcbab4b86d7a488#diff-2abe1226319d07943a47ae2175fd39a81390105905ec26024c89d2a199d00711).

Btw, was the 11 minor version a mistake in the first place? Shouldn't it have been `>=16.0.0 and <=16.15.0`?

Reply:
Arg, good point, I will update it to `16.x.x` for easier groking by people. Which was how it was specified before. I believe the reason it stated `16.11.0` was because that was the first version using `npm >= 8.0.0`.

Reply:
NOTE: I also updated npm to use `8.x.x`. Thanks